subsys/bluetooth/controller/ll_sw/ull_sync_iso.c

```diff
@@ -384,6 +425,30 @@ void ull_sync_iso_setup(struct ll_sync_iso_set *sync_iso,
 	lll->payload_count |= (uint64_t)bi->payload_count_framing[3] << 24;
 	lll->payload_count |= (uint64_t)bi->payload_count_framing[4] << 32;
 
+	if (lll->enc && (bi_size == PDU_BIG_INFO_ENCRYPTED_SIZE)) {
+		const uint8_t BIG3[4] = "3GIB";
+		struct ccm *ccm_rx;
+		uint8_t gsk[16];
```

Comment:
Should the sizes of the `gsk`, `igltk`, etc. be `#define`d somewhere? Perhaps in `include/zephyr/crypto/crypto.h`?

Reply:
I've been adding it when implementing the crypto functions for CSIP: 
```c
#define BT_CSIP_CRYPTO_KEY_SIZE   16
#define BT_CSIP_CRYPTO_SALT_SIZE  16
```

Several other modules have done the same, e.g.
```c
#define ECB_AES_KEY_SIZE   16
#define ECB_AES_BLOCK_SIZE 16
```
and 
```c
#define ATAES132A_AES_KEY_SIZE 16
```

So while `smp.c` doesn't do it, we do it several other places, and I suggest that we do define and use them here as well, as it makes it painfully clear what the individual sizes represent.

Reply:
Most public headers too do not define array size as defines, example `bluetooth.h` (LE SC OOB data), `hci.h`, `crypto.h` etc.. as long as `sizeof()` or `ARRAY_SIZE` can be used, it is redundant to have a define for array size.